### PR TITLE
[Shader Graph] Minor folder move

### PR DESCRIPTION
**Pull request by esmelusina**
This change simply nests the current editor assembly (and associated tests) in a 'Current' subfolder.

This is to make room for additional assemblies that will live alongside 'Current' in the 'Editor' folder. These are going to be introduced in a staging branch and live there for a while, but making this change now on master should help keep things simple.

No new tests are needed.

### CODE REVIEW

**Comment by esmelusina** [comment]
Just waiting on min version from another PR.

File: TestProjects/VisualEffectGraph_HDRP/Assets/ReferenceImages/Linear/WindowsEditor/Direct3D12/None/35_ShaderGraphGenerationFTP.png.meta [hunk 1]
```diff
@@ -80,11 +68,7 @@ TextureImporter:
...
     resizeAlgorithm: 0
     textureFormat: -1
-<<<<<<< HEAD
-    textureCompression: 0
-=======
     textureCompression: 1
->>>>>>> 8a20d46ea1d26dee99c8292d0d16d43d83f7b5e3
     compressionQuality: 50
     crunchedCompression: 0
     allowsAlphaSplitting: 0
```
Comment (jessebarker) [inline TestProjects/VisualEffectGraph_HDRP/Assets/ReferenceImages/Linear/WindowsEditor/Direct3D12/None/35_ShaderGraphGenerationFTP.png.meta:74]: Where did this come from?  Weird merge artifact?  (we didn't really have a meta file with git conflicts in it, did we?)

Comment (esmelusina) [reply TestProjects/VisualEffectGraph_HDRP/Assets/ReferenceImages/Linear/WindowsEditor/Direct3D12/None/35_ShaderGraphGenerationFTP.png.meta:74]: Where is this coming from? Hmm. Weird. I'll make sure those are cleaned up.

**Comment by marctem** [comment]
We'll recreate this in a month after 21.2 branchoff.

**Pull request closed without merging**